views/admin/js/app.js

```diff
@@ -117,9 +117,6 @@ jQuery(document).ready(function($) {
     $(document).on('click', '#btnCreateList', function (e) {
         var name = $('#listName').val();
         var id = $('#clientSelection').val();
-        var optIn = $('#listType').val();

```

Comment:
We don't need to pass the listType information anymore (the element has been removed as well).